pkg/traceroute/flags.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package traceroute
+
+type Args struct {
+	Host      string
+	PacketLen int
+}
+
+type Flags struct {
+	AF4          bool
```

Comment:
I see all these flags and can only remember "think you used enough dynamite there ..."
https://youtu.be/1fM0p-he_Is?si=ukk6K87hAraQQRwY
:-)

Reply:
The flags are taken from the original traceroute and I have to clean that up yet. :D